sample/src/main/java/io/polygon/kotlin/sdk/sample/JavaUsageSample.java

```diff
@@ -108,52 +109,6 @@ public void onError(@NotNull PolygonWebSocketClient client, @NotNull Throwable e
         client.disconnectBlocking();
     }
 
-    public static void websocketLaunchpadSample(String polygonKey) {
```

Comment:
Are you planning on adding a Java sample in a future PR?

Reply:
Good catch. Added a java launchpad example.